platform/kube/inject/http_test.go

```diff
@@ -32,20 +32,21 @@ import (
 )
 
 var (
-	mesh = proxy.DefaultMeshConfig()
-
-	httpTestConfig = &Config{
+	mesh              = proxy.DefaultMeshConfig()
+	unitTestDebugMode = true
```

Comment:
Can you put this with unitTestHub/unitTestTag in platform/kube/inject/inject_test.go?

Reply:
it's not really a constant like the others (I have it set to true in some tests and false in others but sure... not convinced a gloabal for true is useful (like we have `privTrue := true` somewhere, should be `priv := true` or just `true`)
but will change

Reply:
I see what you mean. It looks like `unitTestDebugMode` is only used once in http_test.go? Maybe just remove it altogether.

Reply:
too late, fixed :-) the reason it's in a variable is it is used 3 times (for 2 images and the mode) I wouldn't want to put a literal and have it diverge where only one of the 3 is one way...